src/Build/Instance/TaskRegistry.cs

```diff
@@ -1425,6 +1431,10 @@ private bool GetTaskFactory(TargetLoggingContext targetLoggingContext, ElementLo
                                     return false;
                                 }
                             }
+                            catch(InvalidProjectFileException)
+                            {
+                                throw;
```

Comment:
The new `ProjectErrorUtilities.ThrowInvalidProject()` is in a `try...catch` which wraps the exception again.  This new `catch` is to allow the original exception to be thrown.